packages/dev-utils/src/subproviders/empty_wallet_subprovider.ts

```diff
@@ -0,0 +1,23 @@
+import { JSONRPCRequestPayload } from 'ethereum-types';
+
+import { Callback, ErrorCallback } from './types';
+
+import { Subprovider } from './subprovider';
+
+/**
+ * This is duplicated from Subproviders
+ */
+export class EmptyWalletSubprovider extends Subprovider {
+    // tslint:disable-next-line:prefer-function-over-method async-suffix
+    public async handleRequest(payload: JSONRPCRequestPayload, next: Callback, end: ErrorCallback): Promise<void> {
+        switch (payload.method) {
+            case 'eth_accounts':
+                end(null, []);
```

Comment:
wouldn't you fallback to any other subprovider that may have implemented `eth_accounts` in this case?

Reply:
You only want to use this in testing or when a proper Wallet provider is not available, this will prevent it from reaching the RPC node as it is handled sooner.

INFURA for example will throw on a `eth_accounts`, but you might construct EmptyWallet + INFURA RPC if the user has no metamask.